packages/test/test-service-load/src/workloads/gc/gcDataStores.ts

```diff
@@ -576,7 +580,15 @@ export class SingleCollabDataObject extends BaseDataObject implements IGCActivit
 		}
 
 		this._nodeId = nodeId;
-		this._logger = config.logger;
+		this._logger = createChildLogger({
+			logger: config.logger,
+			namespace: "GC:Stress",
```

Comment:
So what will the full event name look like after this change?

Reply:
`GC:Stress:DS+`. Earlier it was just `DS+`